src/deepsparse/yolact/README.md

```diff
@@ -0,0 +1,186 @@
+
+If no model is specified to the Pipeline for a given task, the Pipeline will automatically select a pruned and quantized model for the task from the SparseZoo that can be used for accelerated inference. Note that other models in the SparseZoo will have different tradeoffs between speed, size, and accuracy.
+
+### DeepSparse Server
+As an alternative to Python API, the DeepSparse Server allows you to serve ONNX models and pipelines in HTTP.
+Both configuring and making requests to the server follow the same parameters and schemas as the
+Pipelines enabling simple deployment.  Once launched, a `/docs` endpoint is created with full
+endpoint descriptions and support for making sample requests.
+
+An example of starting and requesting a DeepSparse Server for YOLACT is given below.
+
+#### Installation
+The Deepsparse Server requirements can be installed by specifying the `server` extra dependency when installing
+DeepSparse.
+
+```bash
+pip install deepsparse[server]
+```
+
+## Deployment Example
+The following example uses pipelines to run a pruned and quantized YOLACT model for inference, downloaded by default from the SparseZoo. 
+As input the pipeline ingests an image (or a list of images) and returns for each image the detection boxes, classification labels, probability scores and segmentation masks. 
+
+[List of the YOLACT SparseZoo Models](
+https://sparsezoo.neuralmagic.com/?domain=cv&sub_domain=segmentation&page=1)
+
+If you don't have an image ready, pull a sample image down with
+
+```bash
+wget -O thailand.jpg https://raw.githubusercontent.com/neuralmagic/deepsparse/main/src/deepsparse/yolact/sample_images/thailand.jpg
+```
+
+```python
+from deepsparse.pipeline import Pipeline
+
+model_stub = "zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none"
+images = ["thailand.jpg"]
+
+yolact_pipeline = Pipeline.create(
+    task="yolact",
+    model_path=model_stub,
+    class_names="coco",
+)
+
+predictions = yolact_pipeline(images=images, confidence_threshold=0.2,nms_threshold = 0.5)
+# predictions has attributes `boxes`, `classes`, `masks` and `scores`
+predictions.classes[0]
+>> ['elephant', 'elephant', 'person', ...]
+
+```
+
+#### Annotate CLI
+You can also use the annotate command to have the engine save an annotated photo on disk.
+```bash
+deepsparse.instance_segmentation.annotate --source thailand.jpg #Try --source 0 to annotate your live webcam feed
+```
+
+Running the above command will create an `annotation-results` folder and save the annotated image inside.
+
+<p align = "center">
+<img src="sample_images/thailand.jpg" alt="original" width="400"/> <img src="sample_images/thailand-annotated.jpg" alt="annotated" width="400"/>
+</p>
+<p align = "center">
+Image annotated with 82.8% sparse and quantized YOLACT
+</p>
+
+If a `--model_filepath` arg isn't provided, then `zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none` will be used by default.
+
+
+#### HTTP Server
```

Comment:
Whats your sense for the most common deployment users do? Is server pretty common use case? Wondering if this HTTP Server section should be the first example under Deployment Examples (cause its super easy to try out!)?

Like 
```
## Deployment examples
<http server>
#### benchmarking
#### annotate script for testing
#### using the pipeline in code
```
?

Reply:
the annotate cli is the most popular use case for the object detection flows - good call out though